packages/webview_flutter/webview_flutter_android/test/webview_android_widget_test.dart

```diff
@@ -843,192 +841,4 @@ void main() {
       verify(mockPlatformHostApi.setWebContentsDebuggingEnabled(false));
     });
   });
-
-  group('WebViewAndroidWebViewClient', () {
```

Comment:
This class no longer exist since we don't need to override `WebViewClient` to set the callback methods.